routers/web/auth/password.go

```diff
@@ -223,7 +228,6 @@ func ResetPasswdPost(ctx *context.Context) {
 		return
 	}
 
-	log.Trace("User password reset: %s", u.Name)
```

Comment:
Unrelated changes. The audit log is not log fro debugging. So I don't know whey this line was removed.